Make.in

```diff
@@ -1,4 +1,5 @@
 # Library name
 -R . CoRN
+-R /home/me/git/math-classes MathClasses
```

Comment:
What is currently the preferred way of doing submodules?
Git-submodules where not preferred by opam.
I'd like something that works for both coq-bench and the coq-CI.
@clarus, any insights?